py-rse/errors.Rmd

```diff
@@ -187,20 +191,17 @@ except ArithmeticError as error:
 Error trying to sum reciprocals: division by zero
 ```
 
-This leads to the rule "throw low, catch high":
+This behavior is designed to support a pattern called "throw low, catch high":
 write most of your code without exception handlers,
 since there's nothing useful you can do in the middle of a small utility function,
-but put a few handlers in the main body of your program
+but put a few handlers in the uppermost functions of your program
 to catch and report all errors.
 
-## What exceptions should I raise and catch? {#errors-where}
-
-Now that we have learned about error handling,
-we can go ahead and add it to our Zipf's Law code.
-First,
-there is already some error handling built in,
-e.g. if you try to read a file that does not exist,
-the error handling inside the `open` function throws a `FileNotFoundError`:
+We can now go ahead and add error handling to our Zipf's Law code.
+Some is already built in:
+for example,
+if we try to read a file that does not exist,
+the `open` function throws a `FileNotFoundError`:
 
 ```shell
 python bin/collate.py results/none.csv
```

Comment:
```suggestion
python bin/collate.py results/none.csv results/dracula.csv
```
Might be a more authentic use of `collate.py` if more than one input file is listed?

Reply:
:+1: